tslearn/clustering.py

```diff
@@ -473,9 +496,16 @@ class TimeSeriesKMeans(BaseEstimator, ClusterMixin,
         Number of iterations for the barycenter computation process. Only used
         if `metric="dtw"` or `metric="softdtw"`.
     metric_params : dict or None
```

Comment:
Perhaps, for consistency reasons, n_jobs should be taken away from metric_params as well and it should be possible to pass this directly?

Reply:
Agree, the docstring for `n_jobs` could mention that it is ignored if the cross-distance matrix cannot be computed using parallelization.